### DISCUSSION TITLE
Multiple exposure calibration outer margin

### DISCUSSION THREAD

**Discussion by dokipen3d** [sn4k3/UVtools#886] [Ideas]
Forgive me if this already exists but I like that there is the 'part margin' for separating the multiple exposures from each other, but is there a margin for the edge of the build plate? Just would be nice to bring them in a bit to make sure there are no issues around the edges (especially with older machines without fresnel that have a slight falloff at the edges.

**Comment by sn4k3** [comment]
There is a global margin. But you can use the tool Move

**Comment by dokipen3d** [comment]
Oh great. where is the global margin setting?

**Comment by dokipen3d** [comment]
Oh wait, I get it now. I previously thought that the green boundary line was the build plate size and that the exposures were always at the edge. My bad, thanks :)
